build/gulp/tasks/bundle.ts

```diff
@@ -88,10 +91,10 @@ task('build:dist:umd', cb => {
   })
 })
 
-task('build:dist', parallel('build:dist:commonjs', 'build:dist:es', 'build:dist:umd'))
-
 // ----------------------------------------
 // Default
 // ----------------------------------------
 
-task('dist', series('clean:dist', 'build:dist'))
+task('bundle', series('bundle:clean', parallel('bundle:build:commonjs', 'bundle:build:es')))
+task('bundle:with-umd', series('bundle', 'bundle:build:umd'))
+task('bundle:all', () => sh('lerna run build'))
```

Comment:
frankly, was a bit lost in naming convention around `bundle` - shouldn't it be `bundle:build:all`? It seems that this will even aid resolution thought on differences between `bundle:with-umd` and `bundle:all`

This might be also appealing that all the places where `bundle:all` is used are, essentially, `build` tasks

Reply:
Renamed:
- `bundle` => `bundle:package`
- `bundle:with-umd` => `bundle:package:with-umd`
- `bundle` => `bundle:all-packages`

Should be much clear now 👍